### DISCUSSION TITLE
How to move the file panel to the right?

### DISCUSSION THREAD

**Discussion by grindarius** [NvChad/NvChad#1983] [Q&A] [answered]
Thank you for this beautiful neovim rice. I would like to know which settings I have to change to get my Files sidebar on the right? Thank you.

**Comment by siduck** [comment, marked as answer]
hi @grindarius , there's a nvimtree option for it, look at their docs

```lua
{ 
 "nvim-tree/nvim-tree.lua",
    opts = {
      view = {
        side = "right",
      },
    }
 }
  ```
  
  add this in your custom plugins

**Reply by grindarius** [reply]
Thank you.
